client/components/view-account.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+export default class ViewAccount extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: props.username,
```

Comment:
Avoid sticking values in state if you can just get them from props whenever they're needed.